# 365 Probabilidades — Dia #012
## Qual a probabilidade de você sair do piloto automático?

**Tipo:** Comportamento  
**Data de publicação:** 2026-06-25  
**Ferramenta:** Python  
**Decisão analisada:** O que eu realmente escolhi — e o que eu só aceitei por inércia?  
**Hashtag:** #365Probabilidades #Dia012

---

### 📖 A História

Tem uma pergunta incômoda que vale a pena se fazer de vez em quando: quanto da minha vida eu de fato escolhi — e quanto eu apenas aceitei, porque era o caminho que já estava lá, pronto, exigindo zero esforço para continuar?

A maioria das pessoas no mundo está no trabalho errado para elas. Não porque não conseguiram nada melhor — muitas vezes porque nunca pararam para revisar se aquele caminho ainda fazia sentido. O medo do desconhecido pesa mais do que o desconforto do conhecido. "Pelo menos eu sei lidar com isso" vira a frase que mantém milhões de pessoas exatamente onde estão.

A ciência comportamental tem um nome para isso: viés do status quo. E a pesquisa mais robusta e atualizada do mundo sobre engajamento no trabalho mostra, com uma precisão impressionante, o tamanho real desse fenômeno.

---

### 📚 O Conceito: Viés do Status Quo

O **viés do status quo**, termo cunhado por Samuelson e Zeckhauser em 1988, descreve a tendência humana de preferir o estado atual das coisas simplesmente porque é o estado atual — não porque foi avaliado e considerado a melhor opção disponível.

Um dos mecanismos centrais por trás desse viés é a aversão à perda combinada com medo do desconhecido: mudar significa arriscar que a próxima situação seja pior, enquanto ficar — mesmo desconfortável — tem o "benefício" da previsibilidade. Esse cálculo, na maioria das vezes, não é feito de forma consciente e deliberada. É simplesmente o caminho de menor resistência.

No mundo do trabalho, isso se manifesta de forma mensurável: a Gallup, organização de pesquisa que acompanha o engajamento de trabalhadores globalmente desde 2009, mede esse fenômeno em escala mundial todos os anos.

---

### 🧮 O Modelo
**Fontes:**
- Gallup (2026) — *State of the Global Workplace: 2026 Report* — dados coletados entre janeiro e dezembro de 2025 — N=263.810 respondentes globais (141.444 empregados), 160 países


In [9]:
import numpy as np
import matplotlib.pyplot as plt
from scipy import stats

plt.rcParams['font.family'] = 'serif'
plt.rcParams['axes.spines.top'] = False
plt.rcParams['axes.spines.right'] = False
plt.rcParams['axes.grid'] = True
plt.rcParams['grid.alpha'] = 0.3
plt.rcParams['figure.facecolor'] = 'white'
plt.rcParams['axes.facecolor'] = 'white'

print('✅ Bibliotecas carregadas')

✅ Bibliotecas carregadas


In [10]:
# --- DADOS DA LITERATURA ---
# Gallup State of the Global Workplace: 2026 Report (dados coletados em 2025)

p_engajados                = 0.20   # empregados genuinamente engajados no trabalho
p_nao_engajados             = 0.64   # nao engajados (presentes, mas desconectados)
p_ativamente_desengajados   = 0.16   # ativamente desengajados (infelizes e demonstram isso)
n_2025                       = 141444  # respondentes empregados em 2025

# Referencia de "melhores praticas" (empresas com cultura de alto engajamento)
p_best_practice = 0.70

fator_correcao = 0.80

print('=' * 65)
print('  DADOS DA LITERATURA — ENGAJAMENTO NO TRABALHO')
print('=' * 65)
print(f'\n  Gallup State of the Global Workplace 2026 (N={n_2025:,} empregados, 160 paises):')
print(f'  → Engajados                    : {p_engajados*100:.0f}%')
print(f'  → Nao engajados                 : {p_nao_engajados*100:.0f}%')
print(f'  → Ativamente desengajados       : {p_ativamente_desengajados*100:.0f}%')
print(f'\n  Referencia de melhores praticas : {p_best_practice*100:.0f}% engajados')
print(f'\n  Fator de correcao aplicado: x{fator_correcao}')
print('=' * 65)

  DADOS DA LITERATURA — ENGAJAMENTO NO TRABALHO

  Gallup State of the Global Workplace 2026 (N=141,444 empregados, 160 paises):
  → Engajados                    : 20%
  → Nao engajados                 : 64%
  → Ativamente desengajados       : 16%

  Referencia de melhores praticas : 70% engajados

  Fator de correcao aplicado: x0.8


In [11]:
# --- O MODELO ---
# Distribuicao Beta com IC 95%

def beta_ic(p, n, conf=0.95):
    alpha = p * n
    beta_  = (1 - p) * n
    dist   = stats.beta(alpha, beta_)
    return dist.interval(conf), dist

ic_engajados,  dist_engajados  = beta_ic(p_engajados, n_2025)
ic_nao_eng,    dist_nao_eng    = beta_ic(p_nao_engajados, n_2025)
ic_ativ_deseng, dist_ativ_deseng = beta_ic(p_ativamente_desengajados, n_2025)

p_engajados_corrigido = p_engajados * fator_correcao
p_desengajados_total  = p_nao_engajados + p_ativamente_desengajados

# Gap entre o mundo real e o cenario de melhores praticas
gap_pp = (p_best_practice - p_engajados) * 100

print('=' * 65)
print('  MODELO — RESULTADO')
print('=' * 65)
print(f'  Engajados:')
print(f'  → Estimativa : {p_engajados*100:.0f}%')
print(f'  → IC 95%     : [{ic_engajados[0]*100:.2f}%, {ic_engajados[1]*100:.2f}%]')
print(f'  → Corrigido  : {p_engajados_corrigido*100:.1f}%')
print(f'\n  Nao engajados:')
print(f'  → Estimativa : {p_nao_engajados*100:.0f}%')
print(f'  → IC 95%     : [{ic_nao_eng[0]*100:.2f}%, {ic_nao_eng[1]*100:.2f}%]')
print(f'\n  Ativamente desengajados:')
print(f'  → Estimativa : {p_ativamente_desengajados*100:.0f}%')
print(f'  → IC 95%     : [{ic_ativ_deseng[0]*100:.2f}%, {ic_ativ_deseng[1]*100:.2f}%]')
print(f'\n  TOTAL DESENGAJADOS (nao engajados + ativamente): {p_desengajados_total*100:.0f}%')
print(f'  GAP para nivel de melhores praticas (70% engajados): {gap_pp:.0f}pp')
print('=' * 65)

  MODELO — RESULTADO
  Engajados:
  → Estimativa : 20%
  → IC 95%     : [19.79%, 20.21%]
  → Corrigido  : 16.0%

  Nao engajados:
  → Estimativa : 64%
  → IC 95%     : [63.75%, 64.25%]

  Ativamente desengajados:
  → Estimativa : 16%
  → IC 95%     : [15.81%, 16.19%]

  TOTAL DESENGAJADOS (nao engajados + ativamente): 80%
  GAP para nivel de melhores praticas (70% engajados): 50pp


In [12]:
# --- VISUALIZACAO ---

# GRAFICO 1 — Distribuicao global de engajamento
fig1, ax1 = plt.subplots(figsize=(12, 8))

categorias = ['Engajados', 'Não engajados', 'Ativamente\ndesengajados']
valores = [p_engajados * 100, p_nao_engajados * 100, p_ativamente_desengajados * 100]
cores1 = ['#2a8a82', '#c8a84b', '#c0392b']

bars = ax1.bar(categorias, valores, color=cores1, alpha=0.85, width=0.5)
ax1.set_ylim(0, 75)
ax1.set_ylabel('Proporção de trabalhadores no mundo (%)', fontsize=12)
ax1.set_title('80% dos trabalhadores do mundo estão desengajados\nGallup State of the Global Workplace 2026 — N=141.444, 160 países',
              fontsize=13, pad=15)

for bar, valor in zip(bars, valores):
    ax1.text(bar.get_x() + bar.get_width() / 2, valor + 1.5,
             f'{valor:.0f}%', ha='center', fontweight='bold', fontsize=16)

plt.tight_layout()
plt.savefig('dia-012-grafico-01-engajamento.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 1 salvo!')


# GRAFICO 2 — O gap para o cenario de melhores praticas
fig2, ax2 = plt.subplots(figsize=(12, 8))

categorias2 = ['Mundo real\n(média global)', 'Melhores práticas\n(cultura de alto engajamento)']
valores2 = [p_engajados * 100, p_best_practice * 100]
cores2 = ['#c0392b', '#2a8a82']

bars2 = ax2.bar(categorias2, valores2, color=cores2, alpha=0.85, width=0.45)
ax2.set_ylim(0, 85)
ax2.set_ylabel('Proporção de engajados (%)', fontsize=12)
ax2.set_title('O tamanho do que fica em aberto quando ninguém revisa o padrão\nGallup (2026) — estimativa de melhores práticas organizacionais',
              fontsize=13, pad=15)

for bar, v in zip(bars2, valores2):
    ax2.text(bar.get_x() + bar.get_width() / 2, v + 2,
             f'{v:.0f}%', ha='center', fontweight='bold', fontsize=16)

ax2.annotate('', xy=(1, p_best_practice*100 - 3), xytext=(0, p_engajados*100 + 3),
             arrowprops=dict(arrowstyle='->', color='#c8a84b', lw=2))
ax2.text(0.5, 45, f'+{gap_pp:.0f}pp\nde espaço', ha='center',
         fontsize=12, color='#c8a84b', fontweight='bold')

plt.tight_layout()
plt.savefig('dia-012-grafico-02-gap.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 2 salvo!')


# GRAFICO 3 — Distribuicao Beta (engajados)
fig3, ax3 = plt.subplots(figsize=(12, 8))

x = np.linspace(ic_engajados[0] * 0.995, ic_engajados[1] * 1.005, 1000)
y = dist_engajados.pdf(x)

ax3.plot(x * 100, y, color='#2a8a82', linewidth=3)
ax3.fill_between(x * 100, y, alpha=0.2, color='#2a8a82')
ax3.axvline(x=ic_engajados[0] * 100, color='#c8a84b', linestyle='--', linewidth=2,
            label=f'IC 95%: [{ic_engajados[0]*100:.2f}%, {ic_engajados[1]*100:.2f}%]')
ax3.axvline(x=ic_engajados[1] * 100, color='#c8a84b', linestyle='--', linewidth=2)
ax3.axvline(x=p_engajados * 100, color='#2a8a82', linewidth=2,
            label=f'Estimativa central: {p_engajados*100:.0f}%')
ax3.axvline(x=p_engajados_corrigido * 100, color='#c0392b', linestyle='-.',
            linewidth=2, label=f'Corrigido (x0.80): {p_engajados_corrigido*100:.1f}%')

ax3.set_xlabel('Probabilidade de estar genuinamente engajado no trabalho (%)', fontsize=12)
ax3.set_ylabel('Densidade', fontsize=12)
ax3.set_title('Distribuição Beta — Engajamento no trabalho\nGallup State of the Global Workplace 2026 — N=141.444',
              fontsize=13, pad=15)
ax3.legend(fontsize=11)

plt.tight_layout()
plt.savefig('dia-012-grafico-03-bayesiano.png', dpi=150, bbox_inches='tight')
plt.close()
print('✅ Grafico 3 salvo!')

✅ Grafico 1 salvo!
✅ Grafico 2 salvo!
✅ Grafico 3 salvo!


### 💡 O Insight

Apenas **20% dos trabalhadores do mundo** estão genuinamente engajados no trabalho. IC 95%: [19,79%, 20,21%] — intervalo extremamente estreito, baseado em 141.444 respondentes em 160 países.

Os outros **80%** se dividem entre não engajados (64%) e ativamente desengajados (16%). Não é uma minoria infeliz. É a maioria absoluta da força de trabalho global, presente todos os dias num lugar que não escolheu — ou escolheu uma vez, há muito tempo, e nunca revisou.

O gap entre essa realidade e o que organizações de alto engajamento conseguem alcançar (cerca de 70%) é de **50 pontos percentuais**. Esse espaço não é preenchido por falta de oportunidade. É preenchido por inércia — pela mesma força que faz alguém manter um plano de previdência só porque foi o padrão, ou manter um emprego só porque trocar dá trabalho e ficar não dá.

*Quanto da carreira, da rotina, da vida que você está vivendo hoje foi resultado de uma escolha ativa — e quanto é só o que ficou, porque nunca foi revisado?*

---

### ⚠️ Limitações do Modelo
- "Engajamento" é definido pela metodologia proprietária da Gallup — outras definições de satisfação no trabalho podem produzir números diferentes
- Os dados são uma média global agregando 160 países com culturas de trabalho muito distintas — a realidade varia significativamente por região
- O número de "melhores práticas" (70%) é uma referência aproximada usada pela própria Gallup, não um padrão científico fixo
- Estar desengajado não significa necessariamente estar na "vida errada" — a causalidade entre status quo e desengajamento é uma interpretação, não uma relação comprovada diretamente pelos dados
- O fator de correção (×0,80) é uma aproximação padrão do projeto

*A ciência é honesta sobre o que não sabe. O modelo também.*

---

### 📎 Links
- Substack: [link do post]
- Instagram: [link do post]

---
*365 Probabilidades · Decidindo com dados, um dia de cada vez.*